Tests/Fixtures/TestProject/Project.xcodeproj/xcshareddata/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -9,7 +9,8 @@
             shouldBeEnabled = "Yes"
             ignoreCount = "0"
             continueAfterRunningActions = "No"
-            scope = "1">
+            scope = "1"
+            stopOnStyle = "0">
```

Comment:
These properties now have a default. Is that correct? What does Xcode do if you create a breakpoint?

Reply:
@yonaskolb 
If I create an Exception Breakpoint,  Xcode set `scope` to `"1"` and `stopOnStyle` to `"0"`.